src/assets/scripts/client/trafficGenerator/buildPreSpawnAircraft.js

```diff
@@ -0,0 +1,163 @@
+import _isEmpty from 'lodash/isEmpty';
+import _isNil from 'lodash/isNil';
+import _isObject from 'lodash/isObject';
+import PositionModel from '../base/PositionModel';
+import RouteModel from '../airport/Route/RouteModel';
+import {
+    fixRadialDist,
+    isWithinAirspace,
+    calculateDistanceToBoundary,
+    bearingToPoint
+} from '../math/flightMath';
+import { nm } from '../utilities/unitConverters';
+
+/**
+ * Loop through `waypointModelList` and determine where along the route an
+ * aircraft should spawn
+ *
+ * @function _calculateSpawnPositions
+ * @param waypointModelList {array<StandardWaypointModel>}
+ * @param spawnOffsets {array}
+ * @return spawnPositions {array}
+ */
+export const _calculateSpawnPositions = (waypointModelList, spawnOffsets, airport) => {
```

Comment:
why the `_` on all helpers in this file? Is it your indication that they are essentially private methods, only intended for use by the main export, `buildPreSpawnAircraft`? If so, why are we still exporting them? Is it maybe the case that we *have* to export these to prevent `buildPreSpawnAircraft()` from throwing "function not defined" errors, because it's in the wrong scope?